docs/topics/lombok.md

```diff
@@ -0,0 +1,149 @@
+Kotlin Lombok compiler plugin should know where this file situates. Use the following syntax in your `build.gradle` file:
+
+```groovy
+kotlinLombok {
+    lombokConfigurationFile(file("lombok.config"))
+}
+```
+
+Learn the [test project with examples of Lombok compiler plugin and `lombok.config` usages](https://github.com/kotlin-hands-on/kotlin-lombok-examples/tree/master/kotlin_lombok_gradle/withconfig).
+
+## Maven
+
+Here's how to use Lombok Kotlin compiler plugin with Maven:
+
+```xml
+<plugin>
+    <groupId>org.jetbrains.kotlin</groupId>
+    <artifactId>kotlin-maven-plugin</artifactId>
+    <version>${kotlin.version}</version>
+    <configuration>
+        <compilerPlugins>
+            <plugin>lombok</plugin>
+        </compilerPlugins>
+        <pluginOptions>
+            <option>lombok:config=${project.basedir}/lombok.config</option>
+        </pluginOptions>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-maven-lombok</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+    </dependencies>
+</plugin>
+```
+
+Please refer to the [Gradle](#gradle) section for detailed information about the Lombok Kotlin compiler plugin configuration.
+
+Learn the [test project example of Lombok compiler plugin and `lombok.config` usages](https://github.com/kotlin-hands-on/kotlin-lombok-examples/tree/master/kotlin_lombok_maven/nokapt).
+
+## Using the plugin along with kapt
+
+If you use [kapt](kapt.md) compiler plugin, note that Java's Lombok should be launched via annotation processor of Java's compiler.
+By default, kapt assumes that all annotation processors work through it and disables Java's compiler's annotation processors.
+To make Lombok working, you need to change this behavior.
+
+If you use the Gradle build system, add the additional flag to the `build.gradle` file to make Java's compiler's annotations
+processors working:
+
+```groovy
+kapt {
+    keepJavacAnnotationProcessors = true
+}
+```
+
+If you use Maven, use the following settings to make Lombok launched by Java's compiler:
+
+```xml
+<plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>3.5.1</version>
+    <configuration>
+        <source>1.8</source>
+        <target>1.8</target>
+        <annotationProcessorPaths>
+            <annotationProcessorPath>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+            </annotationProcessorPath>
+        </annotationProcessorPaths>
+    </configuration>
+</plugin>    
+```
+
+> The Lombok compiler plugin works successfully along with [kapt](kapt.md) if annotations processors don't depend
+> on the code generated by Lombok.
+>
+{type="note"}
+
+Learn the test project examples of kapt and Lombok compiler plugins usages:
```

Comment:
You can use "look through", "see", etc.

Reply:
I thought we try to use only "learn", probably I was mistaken.